### DISCUSSION TITLE
Zapier integration

### DISCUSSION THREAD

**Discussion by tayamueller** [ParabolInc/parabol#5779] [Feedback]
## Issue - Enhancement

from the billing leader of a Pro org w/ 10 active teams:

>"when a card is pushed into Done it could launch a Zapier task to add it to Quip automated tasks; associated to Teams, would be helpful for client work flow"

**Comment by tayamueller** [comment]
feedback from an additional 31-person Pro Org:

> we want to share the after-action report with other teams. But for access or email report everyone would need to be in the other teams, which doesnt make sense. Is there a way to share that semi-publicly? (maybe via API, Zapier etc. would be fine)

> basically, we want to share updates between the other teams and a separate "caretaker" account (see below at zapier use case). - and while CSV is nice, it's not as readable as the mail format.

> we use all the summaries to curate an internal weekly update (like an intranet newsletter), and we started curating the feature announcements from that source as well. 

> - so basically a zapier tasks, that takes all the "done" tickets and puts that into a list
> - and for management things: a zapier tasks, that shows all the stuck issues over all teams

**Comment by jordanh** [comment]
Iceboxing this one for now.

**Comment by jordanh** [comment]
Will reopen on next user request

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/52230202) 🔒 writes:

Message: Hey! Great product! It would be even better if I can schedule meetings creation using Zapier. Honestly webooks for creating/chaning/states/closing meetings would be even better so I can write any integration myself :)
